app/jobs/articles/score_calc_job.rb

```diff
@@ -10,6 +10,8 @@ def perform(article_id)
                              comment_score: article.comments.sum(:score),
                              hotness_score: BlackBox.article_hotness_score(article),
                              spaminess_rating: BlackBox.calculate_spaminess(article))
+
+      article.user&.calculate_score
```

Comment:
It feels a little off that Articles::ScoreCalcJob updating a User's score.

Reply:
Yeah, that makes sense.

Since this should logically be performed _after_ the article's score is calculated, it does make sense that they should occur in a synchronous context.

It could be renamed `ScoreCalcJob` could be renamed to reflect that its job is to calculate all scores related to itself and anything affected by its score, or we create another job sequence when it finishes?

Definitely don't want to over-engineer this.

Reply:
I can live with this for now but I agree it does feel a little off.

Reply:
I think removing the namespace and just calling it `ScoreCalcJob` is a nice middle ground in making it feel less off without being overengineered.